en/installation_guide/install_with_docker_for_contributors.rst

```diff
@@ -11,176 +11,94 @@ Open Orchestra is composed of three projects:
 For contributors, we provide a Docker environment so you get minimal
 setup actions to do.
 
-Install Virtualbox (for MacOS and Windows)
-------------------------------------------
-
-Mac OS
-
-- download http://download.virtualbox.org/virtualbox/4.3.20/VirtualBox-4.3.20-96996-OSX.dmg
-
-Windows
-
-- download http://download.virtualbox.org/virtualbox/4.3.20/VirtualBox-4.3.20-96997-Win.exe
 
 Install Docker
----------------
-The project is running in a virtual environment to be production ready.
-
-### Linux ###
-
-.. code-block:: bash
-
-    $ sudo apt-get update
-    $ sudo apt-get install wget
-    $ sudo apt-get install curl
-    $ wget -qO- https://get.docker.com/ | sh
-    
-Note: If your company is behind a filtering proxy, you may find that the apt-key command fails for the Docker repository during installation. To work around this, add the key directly using the following:
-
-.. code-block:: bash
-
-    $ wget -qO- https://get.docker.com/gpg | sudo apt-key add -
-    
-Create docker group
-
-.. code-block:: bash
-
-    $ sudo usermod -aG docker ubuntu
-	
-Install docker compose
-
-.. code-block:: bash
-
-    $ curl -L https://github.com/docker/compose/releases/download/1.5.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
-    $ chmod +x /usr/local/bin/docker-compose
-    
-### Mac OS ###
-
-.. code-block:: text
-
-    download https://www.docker.com/docker-toolbox (mac version)
-    install the package
-    open a terminal (e.g. http://cmder.net/)
-    
-type the commands
-    
-.. code-block:: bash
-
-    $ docker-machine create default --driver=virtualbox
-    $ eval $(docker-machine env default)
-
-### Windows ###
-
-.. code-block:: text
-
-    download https://www.docker.com/docker-toolbox (mac version)
-    install the package
-    open a terminal (e.g. http://cmder.net/)
-    
-type the commands
-    
-.. code-block:: bash
-
-    $ docker-machine create default --driver=virtualbox
-    $ eval $(docker-machine env default)
-
-Download Composer
------------------
-
-Composer is the package manager used by modern PHP applications.
-
-To install composer with curl:
+--------------
 
-.. code-block:: bash
+Install of Open Orchestra with docker require `Compose 1.6.0+` and `Docker Engine 1.10.+`.
 
-    $ curl -sS https://getcomposer.org/installer | php
+ - Install Docker for `Linux  <https://docs.docker.com/docker-for-mac/>`_
+ - Install Docker for `Mac  <https://docs.docker.com/docker-for-mac/>`_
+ - Install Docker for `Windows  <https://docs.docker.com/engine/installation/linux/>`_
```

Comment:
It looks like you made a small mistake. OS names doesn't match the good OS installation links.